packages/client/wallets/aa/src/SmartWalletSDK.ts

```diff
@@ -27,15 +27,11 @@ export class CrossmintAASDK extends LoggerWrapper {
         this.eaoWalletService = new EOAWalletService(this.crossmintService);
     }
 
-    static init(params: CrossmintAASDKInitParams): CrossmintAASDK {
-        return new CrossmintAASDK(params);
+    static init(params: SmartWalletSDKInitParams): SmartWalletSDK {
+        return new SmartWalletSDK(params);
     }
 
-    async getOrCreateWallet(
-        user: UserIdentifierParams,
-        chain: EVMBlockchainIncludingTestnet,
-        walletConfig: WalletConfig
-    ) {
+    async getOrCreate(user: UserIdentifierParams, chain: EVMBlockchainIncludingTestnet, walletConfig: WalletConfig) {
```

Comment:
this one I may push back on, even if the SDK has wallet in it, still feels more obvious what this does when it says that it creates wallets

Reply:
I added an explicit return type `Promise<EVMSmartWallet>`, does that sway you at all?

Reply:
I think there's too much wallet in `SmartWalletSDK.getOrCreateWallet(...): Promise<EVMSmartWallet>` otherwise

Reply:
reverted this change!